# Try Catch

<p align="center">This branch is made for hometasks</p>

## Tasks

<details>
    <summary> 1. Use try Catch</summary>
    <br>
 Task: How to catch error when function cannot do reverce?

[Solution](./js/reverseString.js)

</details>
<details>
    <summary> 2. Use try Catch</summary>
    <br>
 Task: Complete the isPositive function below. It has one integer parameter a . If the value of a is positive, it must return the string YES. Otherwise, it must throw an Error according to the following rules:

[Solution](./js/yesOrNegative.js)

</details>
<details>
    <summary> 3. Use try Catch</summary>
    <br>
    Task: Convert the code using try...catch:

[Solution](./js/yesOrNegative.js)

</details>
<details>
    <summary> 4. Use try Catch</summary>
    <br>
    Task: The gradeLabs function works for the majority of cases. However, what happens if a student named their function incorrectly? Run gradeLabs and pass it studentLabs2 as defined below..

[Solution](./js/gradeLabs.js)

</details>

## Author

**Arsen Grigoryan**
